pkgs/dart_tooling_mcp_server/test/dart_tooling_mcp_server_test.dart

```diff
@@ -162,72 +191,115 @@ void main() {
       );
     });
 
+    tearDown(() async {
+      // Delete the copy.
+      await Directory(copyDartCliAppsBin).delete(recursive: true);
```

Comment:
nit: I would move this into the `copyDartCliAppAndVerifyContents` function - using `addTearDown`. That way you don't have to remember to do this whenever using that function.